README.md

```diff
@@ -17,6 +17,80 @@ Monorepo of Red Hat Cloud services Components for applications in a React.js env
   * [inventory-compliance](https://github.com/RedHatInsights/frontend-components/tree/master/packages/inventory-compliance#readme) - directly imported component with graphql to show compliance data
   * [inventory-insights](https://github.com/RedHatInsights/frontend-components/tree/master/packages/inventory-insights#readme) - directly imported component to show insights data
 
+## Treeshaking PF with babel plugin
+
+Patternfly packages require some ehancements to be done in order to properly treeshake your bundles. You can either use direct imports or plugin that does that for you, there are actually 2 plugins to do this
+* [babel-plugin-import](https://www.npmjs.com/package/babel-plugin-import) - easy setup, however not that extensible
+* [transform-imports](https://www.npmjs.com/package/transform-imports) - harder to setup, but allows custom rules
+
+Since Patternfly requires a bit of custom settings you should use `transform-imports`. Change your babel to be JS file `babel.config.js` and add these changes to it
+
+```JS
+require.extensions['.css'] = () => undefined;
```

Comment:
I would add a note that is required only in the case of PF. And also that `require.extensions` are deprecated in the latest versions of node. We need to find a different solution fo the future.

Reply:
Good point, I'll add it there.